framework/tlvf/src/include/tlvf/CmduMessageRx.h

```diff
@@ -22,14 +22,14 @@ class CmduParser;
 class CmduMessageRx : public CmduMessage {
 
 public:
-    CmduMessageRx();
-    CmduMessageRx(CmduMessageRx &original);
+    CmduMessageRx() = delete;
+    CmduMessageRx(uint8_t *buff, size_t buff_len);
     ~CmduMessageRx();
-    bool parse(uint8_t *buff, size_t buff_len, bool swap_needed = true, bool parse_tlvs = false);
+    bool parse(bool swap_needed = true, bool parse_tlvs = false);
```

Comment:
It makes perfect sense to move the stuff which should never change to the constructor, such as the buffer and buffer length. The benefit is that this is the only way I could easilly integrate ClassList to CmduMessage - otherwise I would have had to do some pretty ugly adaptations whenever create / parse is called.